source/modes/ConsoleCommands.cpp

```diff
@@ -530,6 +536,11 @@ void addConsoleCommands(ConsoleInterface& cl)
                   cSrvAddCreature,
                   {AbstractModeManager::ModeType::GAME, AbstractModeManager::ModeType::EDITOR});
 
+    // cl.addCommand("startTileCulling",
```

Comment:
Why do you add a command commented ? If it works, just let it be. If not, then just remove it.